projectroles/tests/test_views_ajax.py

```diff
@@ -461,6 +461,431 @@ def test_project_star(self):
+                    'url': f'/project/project/update/{self.project.sodar_uuid}',
+                    'label': 'Update Project',
+                    'icon': 'mdi:lead-pencil',
+                    'active': False,
+                },
+            ],
+        }
+        self.assertEqual(response.json(), expected)
+
+    def test_get_app_links(self):
+        """Test sidebar content retrieval with specific app links"""
+        with self.login(self.user):
+            response = self.client.get(
+                reverse(
+                    'projectroles:ajax_sidebar',
+                    kwargs={'project': self.project.sodar_uuid},
+                )
+                + '?app_name=filesfolders'
+            )
+        self.assertEqual(response.status_code, 200)
+        expected = {
+            'links': [
+                {
+                    'name': 'project-detail',
+                    'url': f'/project/{self.project.sodar_uuid}',
+                    'label': 'Project Overview',
+                    'icon': 'mdi:cube',
+                    'active': False,
+                },
+                {
+                    'name': 'app-plugin-bgjobs',
+                    'url': f'/bgjobs/list/{self.project.sodar_uuid}',
+                    'label': 'Background Jobs',
+                    'icon': 'mdi:server',
+                    'active': False,
+                },
+                {
+                    'name': 'app-plugin-example_project_app',
+                    'url': f'/examples/project/{self.project.sodar_uuid}',
+                    'label': 'Example Project App',
+                    'icon': 'mdi:rocket-launch',
+                    'active': False,
+                },
+                {
+                    'name': 'app-plugin-filesfolders',
+                    'url': f'/files/{self.project.sodar_uuid}',
+                    'label': 'Files',
+                    'icon': 'mdi:file',
+                    'active': True,
+                },
+                {
+                    'name': 'app-plugin-timeline',
+                    'url': f'/timeline/{self.project.sodar_uuid}',
+                    'label': 'Timeline',
+                    'icon': 'mdi:clock-time-eight',
+                    'active': False,
+                },
+                {
+                    'name': 'project-roles',
+                    'url': f'/project/members/{self.project.sodar_uuid}',
+                    'label': 'Members',
+                    'icon': 'mdi:account-multiple',
+                    'active': False,
+                },
+                {
+                    'name': 'project-update',
+                    'url': f'/project/project/update/{self.project.sodar_uuid}',
+                    'label': 'Update Project',
+                    'icon': 'mdi:lead-pencil',
+                    'active': False,
+                },
+            ],
+        }
+        self.assertEqual(response.json(), expected)
+
+    def test_get_no_access(self):
```

Comment:
Permission tests should go in `projectroles.tests.test_permissions_ajax`. Follow the conventions there and, needless to say, ask me if anything is unclear.